### ISSUE TITLE
Usage in Vue 3

### ISSUE THREAD

**Issue by arafatamim** [guanzo/vue-smooth-reflow#24]
Any plans to support Vue 3's new composition API?

**Comment by guanzo** [comment]
Yes. First, I need to learn the composition API and figure out any new "gotchas" with vue 3. For example a vue 3 component doesn't require a single root element, which is an assumption this library makes b/c it was written for vue 2.

**Comment by nickycdk** [comment]
@guanzo , any news on this?

**Comment by mishschmid** [comment]
@guanzo We are wondering the same. Would be nice to keep on using `vue-smooth-reflow` but the use of mixins is a no-go for the latest project unfortunately.

**Comment by KayVeeCodes** [comment]
any news on this?

**Comment by 4Marsupilami** [comment]
@guanzo, any news on this?

**Comment by mreduar** [comment]
Does anyone know of an alternative that works with vue 3?

**Comment by liyasthomas** [comment]
Any update on this? It would be helpful if anyone found an alternative package that supports Vue 3.

**Comment by 4Marsupilami** [comment]
Hi guys.
Here is a nice package for vue 3.
Maintained and developed.

https://auto-animate.formkit.com/

**Comment by mreduar** [comment]
> Hi guys. Here is a nice package for vue 3. Maintained and developed.
> 
> https://auto-animate.formkit.com/

Nice thank you!

**Comment by Nathan7934** [comment]
> Hi guys. Here is a nice package for vue 3. Maintained and developed.
> 
> https://auto-animate.formkit.com/

This can accomplish the same thing, but it is a very over-engineered "no config" solution compared to this library. For example, you cannot specify which properties to animate in formkit's auto-animate. You will likely get a ton of injected animations you don't want even if all you were looking for is a height and width auto animator.

Still looking for a lightweight replacement for this library in vue 3..
